lib/js/__tests__/puppeteer_test.js

```diff
@@ -69,45 +69,54 @@ describe("BrowserFetcher", (function () {
                 browserFetcher[0] = Puppeteer.createBrowserFetcher(Puppeteer$BsPuppeteer.makeBrowserFetcherOptions(undefined, undefined, undefined, /* () */0));
                 return /* () */0;
               }));
+        Jest.testPromise("localRevisions", undefined, (function () {
+                return browserFetcher[0].localRevisions().then((function (revisions) {
+                              revision[0] = Caml_array.caml_array_get(revisions, 0);
+                              return Promise.resolve(Jest.Expect[/* toHaveLength */13](1, Jest.Expect[/* expect */0](revisions)));
+                            }));
+              }));
         Jest.Skip[/* testPromise */2]("canDownload", undefined, (function () {
                 return browserFetcher[0].canDownload("533271").then((function ($$boolean) {
                               return Promise.resolve(Jest.Expect[/* toBe */2](true, Jest.Expect[/* expect */0]($$boolean)));
                             }));
               }));
         Jest.Skip[/* testPromise */2]("download", 30000, (function () {
-                return (function (eta) {
-                              return BrowserFetcher$BsPuppeteer.download("533271", undefined, eta);
-                            })(browserFetcher[0]).then((function (info) {
+                return browserFetcher[0].download("533271", undefined).then((function (info) {
```

Comment:
Don't try to tell me that's not an improvement in the compiled output! Now if we could just do something about that indentation.